src/components/ArchiveSidebar.svelte

```diff
@@ -0,0 +1,85 @@
+<script>
+  import { fly } from "svelte/transition";
+  import {
+    Col,
+    Container,
+    Row,
+    Card,
+    CardHeader,
+    CardBody,
+    Icon,
+  } from "sveltestrap/src";
+  import { stores } from "@sapper/app";
```

Comment:
Let line 12 go before line 2.

This makes the hierarchy of imports clearer since `@sapper` deals with the the backend and frontend, `svelte/...` deals with the frontend, and `sveltestrap/...` is the component library.